README.md

```diff
@@ -0,0 +1,157 @@
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Usage](#usage)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 JavaScript capstone project<a name="about-project"></a>
+
+** JavaScript capstone project ** The JavaScript capstone project is about building web application based on an external API. You will select an API that provides data about a topic that you like and then build the webapp around it. The webapp will have 2 or 3 user interfaces (depending on the size of your team):
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<summary>Client</summary>
+<ul>
+  <li><a href="https://html.com/html5/">HTML5</a></li>
+  <li><a href="https://developer.mozilla.org/en-US/docs/Learn/CSS/First_steps/What_is_CSS">CSS</a></li>
+  <li><a href="https://www.javascript.com/">JavaScript</a></li>
+</ul>
+
+### Key Features <a name="key-features"></a>
+
+- **Add and remove books**
+
+  - The website that displays a list of tasks and allows you to add and remove task from that list.
+
+- **JavaScript Classes**
+
+  - The tasks are managed and manipulated using classes and helper functions within those classes.
+
+- **Store and retrieve data**
+
+  - The displayed tasks are stored and retrieved from the local storage of the browser.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- Visit the live demo [https://gilded-biscochitos-2d3114.netlify.app]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- A browser (e.g. Firefox)
+- An IDE (e.g. [Vususal Studio Code](https://code.visualstudio.com/download))
+- [npm](https://nodejs.org/en/)
+- Terminal
+
+### Setup
+
+To clone this repository to your desired folder, follow the steps below:
+
+**Using the command line**
+
+- Use the following commands
+
+```sh
+  cd my-folder
+  git clone https://github.com/djo1975/JS-CAPSTONE-PP.git
+```
+**How to use tests**
+-Our tests are mede in js using Jest
+-run npm test
+**Using GitHub Desktop app**
+
+- Follow the steps below
+
+  - Visit this link "https://github.com/djo1975/JS-CAPSTONE-PP.git"
+  - Click the green button labelled "code"
+  - Select the "Open with GitHub Desktop" option
+  - After the GitHub Desktop add opens, click the "clone repo" button
+
+### Usage
+
+To run the project, execute the following command:
+
+**Click the "Go Live" button on your IDE(VS Code in this case)**
```

Comment:
[ Optional Changes ]
The previous reviewer did not mention this and it's not going to stop the project from being approved but will help improve your documentation of the hard work you guys did.

- [ ] It would be nice if you could add steps on how one can clone the project, install all packages and be able to start the app locally on their computer. 

For Instance:
- Run `Git clone https://github.com/djo1975/JS-CAPSTONE-PP.git`
- Run `cd JS-CAPSTOPE-PP`
- Run `npm install`
- Run `npm start`